routes/login.js

```diff
@@ -15,6 +15,24 @@ module.exports = function (app, path) {
     });
   });
 
+  /**
+   * GET /account/login/style
+   *
+   * Serves the login page's stylesheet.
+   */
+  app.get("/account/login/style", (req, res) => {
+    res.sendFile(path + "/account/login/style.css");
```

Comment:
Good call, I was about to do the same in the last PR but I wasn't sure how much we'd have to change the login styling since we're probably gonna move to using CSS variables to handle the font and color